client/src/main/java/org/eclipse/hono/client/HonoConnection.java

```diff
@@ -230,10 +230,19 @@ static HonoConnection newConnection(final Vertx vertx, final ClientConfigPropert
      *
      * @return The remote container id or {@code null}.
      */
-    default String getRemoteContainer() {
+    default String getRemoteContainerId() {
         return "N/A";
     }
 
+    /**
+     * Gets the container id as advertised to the peer.
+     * <p>
+     * The identifier consists of the name set in the configuration plus a UUID.
```

Comment:
Is this necessarily the case? I would rather document the requirement regarding the ID's uniqueness instead of describing (and thus defining) how it needs to be created. WDYT?

Reply:
Right, I've adapted the javadoc accordingly.